components/frontapp/actions/import-message/import-message.mjs

```diff
@@ -49,20 +50,17 @@ export default {
       type: "string",
       description: "Body of the message.",
     },
-    body_format: {
-      type: "string",
-      description: "Format of the message body. Ignored if the message type is not email. Can be one of: 'html', 'markdown'. (Default: 'markdown')",
-      optional: true,
-      options: [
-        "html",
-        "markdown",
+    bodyFormat: {
+      propDefinition: [
+        frontApp,
+        "bodyFormat",
       ],
     },
-    external_id: {
+    externalId: {
       type: "string",
       description: "External identifier of the message. Front won't import two messages with the same external ID.",
     },
-    created_at: {
+    createdAt: {
```

Comment:
Can this be a `optional` prop with default as `new Date().getTime()`?

Reply:
Actually is a required field